librosa/sequence.py

```diff
@@ -359,6 +361,341 @@ def __dtw_backtracking(D_steps, step_sizes_sigma):  # pragma: no cover
     return wp
 
 
+def rqa(sim, gap_onset=1, gap_extend=1, knight_moves=True, backtrack=True):
+    '''Recurrence quantification analysis (RQA)
+
+    This function implements different forms of RQA as described by
+    Serra, Serra, and Andrzejak [1]_.  These methods take as input
+    a recurrence or cross-similarity matrix `sim`, and calculate the value
```

Comment:
~what is the difference between a recurrence matrix and a cross-similarity matrix?~
also note that, because "recurrence" is a noun, it might not be obvious to non-native English speakers that it is adjectived and that the noun "matrix" applied to both "recurrence" or "cross-similarity".

Reply:
bis: i would be in favor of replacing recurrence by self-similarity

Reply:
I went with recurrence to match the function name, but we have a see-also, so i'm not too picky about it.